hound_test.rb

```diff
@@ -0,0 +1,7 @@
+puts 'this is a violation'
+_things = { one: 1 }
+puts 'this is a violation'
+puts 'this is a violation'
+puts 'this is a violation'
+puts 'this is a violation'
+puts 'this is a violation'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.